### DISCUSSION TITLE
How do I open folders?

### DISCUSSION THREAD

**Discussion by yasminlovescranberrys** [d0k3/GodMode9#821] [Q&A]
Hi, I have a file that i downloaded in a 7-zip folder. And I put the entire folder on my SD-card. But my problem now is, that i can’t look inside the folder to see the files inside. I’ve tried to see inside it with Godmode9 and Fbi but nothing works. Am I missing something?

**Comment by Epicpkmn11** [comment]
You need to extract the 7z file with something first, I think [3DShell](https://db.universal-team.net/3ds/3dshell) can do that on console
